### DISCUSSION TITLE
Combined manual & agent instrumentation

### DISCUSSION THREAD

**Discussion by nebffa** [open-telemetry/opentelemetry-java-instrumentation#10539] [Q&A] [answered]
Similar discussions are here:
https://github.com/open-telemetry/opentelemetry-java/discussions/4485
https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/6005

We are trying to use agent instrumentation alongside manual instrumentation. We've taken care to make sure we're using the same version of the opentelemetry-sdk between our manual instrumentation and the agent.

Our versions are:
agent: 1.32.1 (which uses 1.34.1 of the SDK)
sdk/api: 1.34.1

Yet when we emit manual spans, they are not attached to the agent traces. We are acquiring the tracer via `GlobalOpenTelemetry.get().getTracer()`. Is there anything special that needs to be done?

**Comment by laurit** [comment]
Manual instrumentation should work nicely together with the automatic instrumentation. Please provide a minimal sample application that reproduces your issue.

**Reply by nebffa** [reply, marked as answer]
I did a bit more investigation and I found that combined tracing works well with HTTP traces, but was not working with consuming messages from SQS. I'll have to look into that further, but it's good to know that the high-level functionality is working
